shortcut_composer/api_krita/enums/action.py

```diff
@@ -0,0 +1,581 @@
+# SPDX-FileCopyrightText: © 2022 Wojciech Trybus <wojtryb@gmail.com>
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from krita import Krita as Api
+from enum import Enum
+
+from PyQt5.QtGui import QIcon
+
+
+class Action(Enum):
+    """
+    Contains all known actions of Krita.
+
+    Example usage: `Tool.FREEHAND_BRUSH`
+
+    Available actions:
+    - `TBA`
+    """
+
+    # Filters
+    FILTER_APPLY_AGAIN = "filter_apply_again" # Apply Filter Again
```

Comment:
Konwencja `enum = nazwa_w_kricie #komentarz` mi się nie podoba.

Środkowa informacja jest do komunikacji z kritą, prawa z użytkownikiem i programistą. Jeżeli komentarz jest potrzebny, to znaczy, że nazwa enuma jest niedeskryptywna i zła. Powielanie informacji też nic nie daje.

Trzymałbym się nazw jakie akcje mają w `Configure Shortcuts`, bo po takich nazwach użytkownik będzie ich szukał. Jeżeli część informacji wynika z nazwy enuma, wtedy dla mnie jest akceptowalne je wywalić.

Jeżeli nazwa w kricie jest mocno niejasna i wymaga komentarza, dopiero wtedy bym to zmieniał nazwę enuma licząc, że bardziej pomoże się komuś to znaleźć niż zaszkodzi.

Reply:
Thx! No na razie wygenerowałem Enumy z `action.objectName()` a komentarze z `action.text()` (to, co widać w `Configure Shortcuts`). Myślę, że to odwrócę i potem ręcznie poprawie niektóre Enumy jak planowałem, najpierw chciałem dostać feedback, zanim się narobię xD

Podeślę potem wersje z `action.text()` jako nazwą Enumów, jak dasz zielone światło, to ruszę z ręcznym poprawianiem tak jak to opisałeś.